default/python/tox.ini

```diff
@@ -2,6 +2,8 @@
 extend-ignore =
     E203,E501,W503,W504,E121,E126,E226  # handled by black
 
+max-complexity = 10
```

Comment:
To be honest, I do not like a hard limit of 10. During development I often check in more complex functions only to split them up once I figured out what the code should do. With AI development this is not always trivial.

Can we agree on 15 here? That doesn't stop us from aiming lower, but I don't want to keep adding noqas.

Btw. #4137 is still waiting for your approval.

Reply:
Lets try to keep it a 10 for a while.

If this number will lead to too many suppressions we will update it.  
Now we have 64 suppressions, and we could get back to this after we will have more than 70. 

During development, you could just commit with `-n` / `--no-verify`.
I usually try to commit as is and fix other warnings reported by pre-commit. 

When you finished you could run eigher `flake8` or `pre-commit run --all-files`